diff --git a/package/libs/openssl/Config.in b/package/libs/openssl/Config.in
index ad2396df0b..50f16c4d8f 100644
--- a/package/libs/openssl/Config.in
+++ b/package/libs/openssl/Config.in
@@ -29,7 +29,7 @@ config OPENSSL_SMALL_FOOTPRINT
 config OPENSSL_KTLS
 	bool
 	prompt "Enable kTLS support"
-	select PACKAGE_kmod-tls
+	depends on PACKAGE_kmod-tls
 	help
 		This will enable kTLS support, allowing data encryption
 		operations to be performed in kernel space.
