--- a/feeds/luci/modules/luci-base/ucode/dispatcher.uc
+++ b/feeds/luci/modules/luci-base/ucode/dispatcher.uc
@@ -968,7 +968,7 @@
 
 					// Show login form with 2FA fields if required
 					let scope = {
-						duser: 'root',
+						duser: 'admin',
 						fuser: user,
 						auth_fields: auth_fields,
 						auth_message: auth_message,
@@ -1013,7 +1013,7 @@
 						http.header('X-LuCI-Login-Required', 'yes');
 
 						let scope = {
-							duser: 'root',
+							duser: 'admin',
 							fuser: user,
 							auth_plugin: length(auth_check.challenges) ? auth_check.challenges[0].name : null,
 							auth_fields: auth_check.fields,
