--- a/nss-setup/config-nss.seed
+++ b/nss-setup/config-nss.seed
@@ -38,7 +38,7 @@ CONFIG_TARGET_qualcommax=y
 # Target device
 #####################################################################
 # Uncomment target device you want to build for, set '=y'
-CONFIG_TARGET_qualcommax_ipq807x=y
+# CONFIG_TARGET_qualcommax_ipq807x is not set
 
 # CONFIG_TARGET_qualcommax_ipq807x_DEVICE_aliyun_ap8220 is not set
 # CONFIG_TARGET_qualcommax_ipq807x_DEVICE_arcadyan_aw1000 is not set
@@ -81,7 +81,7 @@ CONFIG_TARGET_qualcommax_ipq807x=y
 # UNTESTED
 
 # Uncomment the following line to build for IPQ60xx devices.
-# CONFIG_TARGET_qualcommax_ipq60xx=y
+CONFIG_TARGET_qualcommax_ipq60xx=y
 
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_8devices_mango-dvk is not set
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_alfa-network_ap120c-ax is not set
@@ -90,7 +90,7 @@ CONFIG_TARGET_qualcommax_ipq807x=y
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_glinet_gl-ax1800 is not set
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_glinet_gl-axt1800 is not set
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_linksys_mr is not set
-# CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_linksys_mr7350 is not set
+CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_linksys_mr7350=y
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_linksys_mr7500 is not set
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_netgear_wax214 is not set
 # CONFIG_TARGET_qualcommax_ipq60xx_DEVICE_netgear_wax610-common is not set
